00-RELEASENOTES

```diff
@@ -9,7 +9,7 @@ Usually "unstable" is stable enough for you to use it in development environment
 however you should never use it in production environments. It is possible
 to download the latest stable release here:
 
-    http://download.redis.io/releases/redis-stable.tar.gz
+    https//download.redis.io/redis-stable.tar.gz
```

Comment:
```suggestion
    https://download.redis.io/redis-stable.tar.gz
```

Reply:
LOL.. i swear i copied that link to a browser to test that it works.
:man_shrugging: